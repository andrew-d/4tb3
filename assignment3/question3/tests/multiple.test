<TABLE><TR><TD>foobar</TD><TD>another</TD></TR><TR><TD>row2</TD><TD>corner</TD></TR></TABLE>
