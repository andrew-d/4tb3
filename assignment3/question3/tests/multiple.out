| foobar  | another |
| row2    | corner  |
